src/wp-includes/block-editor.php

```diff
@@ -694,3 +673,40 @@ function get_block_editor_theme_styles() {
 
 	return $styles;
 }
+
+/**
+ * Returns the legacy theme supports settings for block editor.
+ *
+ * @since 6.2.0
+ *
+ * @return array The legacy theme supports settings.
+ */
+function get_legacy_theme_supports_block_editor_settings() {
```

Comment:
Bit of a nit-pick: Isn't this about classic themes? Would it make sense to call it `get_classic_theme_supports_block_editor_settings()` instead? Or what does the "legacy" refer to?

Reply:
+1, `classic_theme` would be a better prefix than `legacy` IMO :)

Reply:
And obviously, same goes for the docblock, and of course the function calls from `class-wp-theme-json-resolver.php` ;)